plugin/adsk/plugin/exportTranslator.h

```diff
@@ -16,16 +16,18 @@
 //
 #pragma once
 
-#include "base/api.h"
-#include <mayaUsd/fileio/jobs/jobArgs.h>
+#include <mayaUsdPlugin/base/api.h>
 
-#include "pxr/pxr.h"
+#include <mayaUsd/base/api.h>
+#include <mayaUsd/fileio/jobs/jobArgs.h>
```

Comment:
I actually consider this **7. Your project’s headers** - I was considering pretty much anything in maya-usd as in "this project" - but I'm guessing you're considering this **4. Other libraries’ headers**?

We should probably come to a decision about this - from a practical standpoint, I strongly prefer going with the anything-in-maya-usd interpretation, since that can be easily enforced via a single .clang-format file, while doing different things in different sub-directories will be much trickier to implement.

Plus, I like the simplicity of it.  And, it's more thematically in keeping with the "more general to more specific" ordering - things within the same project are more specific than general maya or pixar headers.